### PR TITLE
Parallel psim xf

### PR DESCRIPTION
**Pull request by dylanwells37**
This pull request will run peuoSim jobs for each antenna after the XF job finishes.
Documentation here: [http://radiorm.physics.ohio-state.edu/elog/GENETIS/234](http://radiorm.physics.ohio-state.edu/elog/GENETIS/234)

Note:
Maybe combine the XF GPU jobs into only num_keys jobs and create a flag system before finishing the pull request?
I'll try to see if it will work this week.

### CODE REVIEW

**Review by ezio-melotti — COMMENTED** [review]
The changes to the Python code look ok to me.

File: BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_B/Part_B2_Parallel_Pueo.sh [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+	then
+	    if [ $already_checked -eq 0 ]
+	    then
+			xf_finish_time=`date +%s`
+			pueosim_start_time=`date +%s`
+			already_checked=1
+		fi
+	fi
+
+	tput cuu 3
```
Comment (Machtay) [inline BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_B/Part_B2_Parallel_Pueo.sh:146]: Just out of curiosity, what does this do? Looks like maybe it puts me back to the top of my terminal when I run it on my own terminal.

Comment (dylanwells37) [reply BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_B/Part_B2_Parallel_Pueo.sh:146]: It's so the progress bar I made doesn't move when printing it

Comment (Machtay) [reply BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_B/Part_B2_Parallel_Pueo.sh:146]: Gotcha, everything looks good to me then!

**Pull request merged**